lib/web_ui/dev/firefox_installer.dart

```diff
@@ -188,16 +195,20 @@ class FirefoxInstaller {
     ));
 
     final io.File downloadedFile =
-        io.File(path.join(versionDir.path, 'firefox-${version}.tar.bz2'));
-    await download.stream.pipe(downloadedFile.openWrite());
+        io.File(path.join(versionDir.path, PlatformBinding.instance.getFirefoxDownloadFilename(version)));
+    io.IOSink sink = downloadedFile.openWrite();
```

Comment:
Thanks! So if we don't do this, we are leaking.

I think we made this error in a few more places :) https://github.com/flutter/engine/blob/master/lib/web_ui/dev/chrome_installer.dart#L190

I'll fix them!

Reply:
Thank you!